animation_nodes/nodes/rotation/c_utils.pyx

```diff
@@ -121,3 +122,139 @@ def randomQuaternionList(int seed, int amount):
         quaternionNormalize_InPlace(result.data + i)
 
     return result
+
```

Comment:
As we said in the original pull request. I want `cdef` functions to do each of the conversions in the math module. Then those functions can be used here.

Reply:
OK, let me try.